src/main/scala/viper/gobra/frontend/Config.scala

```diff
@@ -116,9 +136,31 @@ class ScallopGobraConfig(arguments: Seq[String], isInputOptional: Boolean = fals
     */
   val input: ScallopOption[List[String]] = opt[List[String]](
     name = "input",
-    descr = "List of Gobra programs or a single package name to verify"
+    descr = "List of Gobra programs, a project directory or a single package name to verify",
+    short = 'i'
   )
 
+  val recursive: ScallopOption[Boolean] = toggle(
+    name = "recursive",
+    descrYes = "Perform verification on all subfolders of the specified input directory",
+    default = Some(false),
+    short = 'r'
+  )
+
+  val packages: ScallopOption[List[String]] = opt[List[String]](
+    name = "packages",
+    descr = "Which packages should be verified, all packages are verified per default",
```

Comment:
What do you mean by "all packages are verified per default"?

Reply:
The option should limit, which packages are verified. If it is omitted, all packages found in the specified inputs are verified